packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -244,8 +244,8 @@ export const ChatScreen = () => {
   const popupMessage = useSelector((state) =>
     getChatMessageById(state, chatId ?? '', popupMessageId ?? '')
   )
-  const { canSendMessage, firstOtherUser: otherUser } =
-    useCanSendMessage(chatId)
+  const { canSendMessage, firstOtherUser } = useCanSendMessage(chatId)
+  const otherUser = firstOtherUser ?? {}
```

Comment:
Is this better than leaving undefined? I think later in the file I am accessing `otherUser.user_id` conditionally on `!!otherUser`.

Reply:
interesting, if `otherUser` is undefined the component fails to render due to https://github.com/AudiusProject/audius-client/pull/3514/files#diff-599f9b15aa87cbaec942aaf03242204cf8cd103687bde3f1416bc0e039af3a68R287